luigi/file.py

```diff
@@ -45,6 +47,7 @@ def generate_tmp_path(self, path):
 
 
 class LocalFileSystem(FileSystem):
+
```

Comment:
Are these really pep8 fixes??? I mean, you didn't change this code and the pep8 check (`tox pep8`) ran fine before. Hmm...